WorkspaceServer/Packaging/WebAssemblyAssetFinder.cs

```diff
@@ -37,12 +38,18 @@ async Task<TPackage> IPackageFinder.Find<TPackage>(PackageDescriptor descriptor)
         protected async Task<IPackage> CreatePackage(PackageDescriptor descriptor, PackageTool tool)
         {
             await tool.Prepare();
-            var wasmAsset = await tool.LocateWasmAsset();
-            if (wasmAsset != null)
+            var buildAsset = await tool.LocateProjectAsset();
+            if (buildAsset != null)
             {
-                var package = new Package2(descriptor.Name, new FileSystemDirectoryAccessor(wasmAsset.DirectoryAccessor.GetFullyQualifiedRoot().Parent));
-                package.Add(wasmAsset);
-                return package;
+                var package = new Package2(descriptor.Name, new FileSystemDirectoryAccessor(buildAsset.DirectoryAccessor.GetFullyQualifiedRoot().Parent));
```

Comment:
>new FileSystemDirectoryAccessor(buildAsset.DirectoryAccessor.GetFullyQualifiedRoot().Parent) [](start = 60, length = 92)

Moving to a parent directory from the asset seems like a can of worms that I feel breaks the abstraction.

We have a method, DirectoryAccessor.GetDirectoryAccessorForRelativePath, which I feel we should be using here. WebAssemblyAssetFinder should take an IDirectoryAccessor in its constructor and build one accessor from another so when we test it via the in-memory implementation, it won't break out into a file-based implementation.